flow/connectors/mysql/cdc.go

```diff
@@ -393,6 +394,11 @@ func (c *MySqlConnector) PullRecords(
 		otelManager.Metrics.FetchedBytesCounter.Add(ctx, int64(len(event.RawData)))
 
 		switch ev := event.Event.(type) {
+		case *replication.GTIDEvent:
+			if ev.ImmediateCommitTimestamp > 0 {
+				otelManager.Metrics.CommitLagGauge.Record(ctx,
+					time.Now().UTC().Sub(time.UnixMicro(int64(ev.ImmediateCommitTimestamp))).Microseconds())
```

Comment:
Will need to initialize the gauge before using it.
I have an issue opened to add tests to fail in case we miss initializing #2968